simpleble/CMakeLists.txt

```diff
@@ -135,7 +135,12 @@ target_link_libraries(simpleble-c PRIVATE simpleble::simpleble)
 append_sanitize_options("${SIMPLEBLE_SANITIZE}")
 
 if(NOT SIMPLEBLE_LOG_LEVEL)
-set(SIMPLEBLE_LOG_LEVEL "INFO")
+    set(SIMPLEBLE_LOG_LEVEL "INFO")
+endif()
+
+
+if(SIMPLEBLE_USE_SESSION_DBUS)
+    add_definitions (-DUSE_SESSION_DBUS)
```

Comment:
Using `add_definitions` without specifying a target will also propagate to any user consuming this library.

Instead, use:
```
list(APPEND PRIVATE_COMPILE_DEFINITIONS SIMPLEBLUEZ_USE_SESSION_DBUS)
```

and place it inside the Linux part of the settings, as it only applies to that OS.